mobile-browser-based-version/src/store/store.js

```diff
@@ -0,0 +1,29 @@
+import {createStore} from 'vuex'
```

Comment:
this file needs to be updated as we discussed

Reply:
The issue is resolved, the new code is in the file MainTaskFrame and not GlobalTaskFrame 👍